docs/src/_parts/commands/k8s_refresh-certs.md

```diff
@@ -0,0 +1,21 @@
+## k8s refresh-certs
+
+Refresh the certificates of the running node
+
+```
+k8s refresh-certs [flags]
+```
+
+### Options
+
+```
+      --extra-sans stringArray   extra SANs to add to the certificates.
+  -h, --help                     help for refresh-certs
+      --timeout duration         the max time to wait for the command to execute (default 1m30s)
+      --ttl string               the time-to-live for the certificates.
```

Comment:
Two things:
* I think we don't use abbreviations in the CLI (IIRC there was a long discussion about it ~a year ago). For that reason I used `--expires-in` for #620. Maybe we should do that here as well, wdyt?
* I would expect this to be a `time.Duration`. Any reason for this to be a string?

Reply:
- That sounds great, I will change the flag name from `--ttl` to `--expires-in`
- Please see this comment for more context: https://github.com/canonical/k8s-snap/pull/623#discussion_r1731419067

Reply:
ack